articles/quickstart/spa/react-beta/02-calling-an-api.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Calling an API
+description: This tutorial demonstrates how to make API calls for protected resources on your server.
+budicon: 546
+topics:
+  - quickstarts
+  - spa
+  - react
+  - apis
+github:
+  path: 02-Calling-an-API
+sample_download_required_data:
+  - client
+  - api
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD041 MD002 -->
+
+Most single-page apps use resources from data APIs. You may want to restrict access to those resources, so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](/api-auth).
```

Comment:
```suggestion
Most single page applications use resources from data APIs. You may want to restrict access to those resources, so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](/api-auth).
```